wellcomecollection/wellcomecollection.org
# Highlight tour stop page

For #11037 

## What does this change?
Adds a highlight tour stop page at `guides/exhibitions/[id]/[type]/[stop]`

<img width="503" alt="image" src="https://github.com/user-attachments/assets/4a4295c6-7211-4864-8bb5-68ffa3cdf88a">


## How to test

- run locally fetching data from Prismic stage (the toggle doesn't work for this – you have to update the `getRepositoryUrl` to `wellcomecollecction-stage` manually)
- visit http://localhost:3000/guides/exhibitions/ZrHvtxEAACYAWmfc/audio-without-descriptions/2
- check the audio player works, the close cross returns you to the list of stops, the prev/next links take you to the appropriate stop
- change guide type and check the [video version](http://localhost:3000/guides/exhibitions/ZrHvtxEAACYAWmfc/audio-without-descriptions/2) also works 

## How can we measure success?
Validate this is a better ux journey for exhibition guide stops

## Have we considered potential risks?
We're doing more fetching than we need to from Prismic the first time you go to any of the stop pages _or_ the stops overview page, you have all the data you need. I started looking at [swr](https://swr.vercel.app/) as a way to avoid doing this, but I _think_ we should be able to solve it using the `shallow` prop on a `NextLink`. I've started looking at this, but I think getting it behind a toggle to test the journey should take precedence
